node/src/chain_spec/oak.rs

```diff
@@ -196,6 +197,115 @@ pub fn oak_staging() -> ChainSpec {
 	)
 }
 
+pub fn oak_live() -> ChainSpec {
+	// Give your base currency a unit name and decimal places
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("tokenSymbol".into(), TOKEN_SYMBOL.into());
+	properties.insert("tokenDecimals".into(), TOKEN_DECIMALS.into());
+	properties.insert("ss58Format".into(), SS_58_FORMAT.into());
+
+	ChainSpec::from_genesis(
+		// Name
+		"OAK Network",
+		// ID
+		"oak",
+		ChainType::Live,
+		move || {
+			let allocation_json = &include_bytes!("../../../distribution/oak_alloc.json")[..];
+			let initial_allocation: Vec<(AccountId, Balance)> =
+				serde_json::from_slice(allocation_json).unwrap();
+			const ALLOC_TOKENS_TOTAL: u128 = DOLLAR * 1_000_000_000;
+			validate_allocation(
+				initial_allocation.clone(),
+				ALLOC_TOKENS_TOTAL,
+				EXISTENTIAL_DEPOSIT,
+			);
+
+			let vesting_json = &include_bytes!("../../../distribution/oak_vesting.json")[..];
+			let initial_vesting: Vec<(u64, Vec<(AccountId, Balance)>)> =
+				serde_json::from_slice(vesting_json).unwrap();
+
+			let vested_tokens = 9_419_999_999_999_999_919;
+			let vest_starting_time: u64 = 1669874400;
+			let vest_ending_time: u64 = 1764568800;
+			validate_vesting(
+				initial_vesting.clone(),
+				vested_tokens,
+				EXISTENTIAL_DEPOSIT,
+				vest_starting_time,
+				vest_ending_time,
+			);
+
+			testnet_genesis(
+				// initial collators.
+				vec![
+					(
+						// SS58 prefix substrate: 5EvKT4iWQ5gDnRhkS3d254RVCfaQJSyXHj5mA4kQTsCjWchW
+						hex!["7e4f4efde71551c83714fb3062724067e7bb6ebc3bf942813321f1583e187572"]
+							.into(),
+						hex!["7e4f4efde71551c83714fb3062724067e7bb6ebc3bf942813321f1583e187572"]
+							.unchecked_into(),
+					),
+					(
+						// SS58 prefix substrate: 5F4Dx9TD16awU7FeGD3Me5VDrEsL5MDPEcbNBvUgLQFawxyW
+						hex!["8456c30af083c2b2d767d6950e8ee654d4eff5e69cdd9f75db5bcbf2d7b0d801"]
+							.into(),
+						hex!["8456c30af083c2b2d767d6950e8ee654d4eff5e69cdd9f75db5bcbf2d7b0d801"]
+							.unchecked_into(),
+					),
+				],
```

Comment:
Any reason to not include all 4 four of the initial collators that we plan to launch with?
Also, double checking that these are new accounts (not used on other networks) and that someone has the secret seed stored. We will need to provide the seed to the collator nodes for the chain to start producing.

Reply:
These are 4 new accounts, yes